smart-contracts/wasm-chain-integration/src/v1/mod.rs

```diff
@@ -25,6 +25,11 @@ use wasm_transform::{
     utils,
 };
 
+// TOOD: Move these to concordium-contracts-common?
```

Comment:
I have added a module reference in concordium-contracts-common since I also needs it.
Will make a PR soon

Reply:
https://github.com/Concordium/concordium-contracts-common/pull/55